targets/TARGET_TI/TARGET_MSP432/TARGET_MSP432P401R/device/TOOLCHAIN_ARM_STD/msp432p401r.sct

```diff
@@ -0,0 +1,42 @@
+#! armcc -E
+
+/* Device specific values */
+
+#define ROM_START   0x00000000
+#define ROM_SIZE    0x00040000
+#define RAM_START   0x20000000
+#define RAM_SIZE    0x00010000
+#define STACK_SIZE  0x00000400
+#define VECTORS     0
+//#define VECTORS     57   /* This value must match NVIC_NUM_VECTORS */
+
```

Comment:
It might also be possible to simply not define it, I will try that.
With MSP432 I prefer to have the vector table at the beginning of the Flash (no copying to start of RAM). I hope this is okay.

Reply:
I removed the whole VECTORS stuff in the last commit!